runelite-client/src/test/java/net/runelite/client/plugins/prayer/PrayerPluginTest.java

```diff
@@ -0,0 +1,185 @@
+public class PrayerPluginTest
+{
+
+	@Inject
+	private PrayerPlugin prayerPlugin;
+
+	@Mock
+	@Bind
+	private Client client;
+
+	@Mock
+	@Bind
+	private PrayerConfig config;
+
+	@Mock
+	@Bind
+	private OverlayManager overlayManager;
+
+	@Mock
+	@Bind
+	private InfoBoxManager infoBoxManager;
+
+	@Mock
+	@Bind
+	private SpriteManager spriteManager;
+
+	@Mock
+	@Bind
+	private ScheduledExecutorService executor;
+
+	@Mock
+	@Bind
+	private ItemManager itemManager;
+
+	@Before
+	public void before()
+	{
+		Guice.createInjector(BoundFieldModule.of(this)).injectMembers(this);
+	}
+
+	@Test
+	public void testGetEstimatedTimeRemainingOverOneHour()
+	{
+
+		String expectedString = "1:19:12";
+
+		ItemStats itemStats = new ItemStats(false, true, 1, 8,
+			ItemEquipmentStats.builder()
+				.slot(EquipmentInventorySlot.WEAPON.getSlotIdx())
+				.prayer(50)
+				.build());
+
+		ItemContainer itemContainer = mock(ItemContainer.class);
+
+		when(client.isPrayerActive(Prayer.PRESERVE)).thenReturn(true);
+		when(client.getBoostedSkillLevel(Skill.PRAYER)).thenReturn(99);
+		when(client.getItemContainer(InventoryID.EQUIPMENT)).thenReturn(itemContainer);
+		when(itemContainer.getItems()).thenReturn(new Item[]{new Item(ItemID.TWISTED_BOW, 1)});
+		when(itemManager.getItemStats(anyInt(), anyBoolean())).thenReturn(itemStats);
+
+		prayerPlugin.onItemContainerChanged(new ItemContainerChanged(InventoryID.EQUIPMENT.getId(), itemContainer));
+		String actualString = prayerPlugin.getEstimatedTimeRemaining(false);
+
+		Assert.assertEquals(expectedString, actualString);
+	}
+
+	@Test
+	public void testGetEstimatedTimeRemainingUnderOneHour()
+	{
+
+		String expectedString = "29:42";
+
+		ItemContainer itemContainer = mock(ItemContainer.class);
+
+		when(client.isPrayerActive(Prayer.PRESERVE)).thenReturn(true);
+		when(client.getBoostedSkillLevel(Skill.PRAYER)).thenReturn(99);
+		when(client.getItemContainer(InventoryID.EQUIPMENT)).thenReturn(itemContainer);
+		when(itemContainer.getItems()).thenReturn(new Item[]{});
+
+		prayerPlugin.onItemContainerChanged(new ItemContainerChanged(InventoryID.EQUIPMENT.getId(), itemContainer));
+		String actualString = prayerPlugin.getEstimatedTimeRemaining(false);
+
+		Assert.assertEquals(expectedString, actualString);
+	}
+
+	@Test
+	public void testGetEstimatedTimeRemainingFormatForOrbUnderOneHour()
+	{
+
+		String expectedString = "29m";
+
+		ItemContainer itemContainer = mock(ItemContainer.class);
+
+		when(client.isPrayerActive(Prayer.PRESERVE)).thenReturn(true);
+		when(client.getBoostedSkillLevel(Skill.PRAYER)).thenReturn(99);
+		when(client.getItemContainer(InventoryID.EQUIPMENT)).thenReturn(itemContainer);
+		when(itemContainer.getItems()).thenReturn(new Item[]{});
+
+		prayerPlugin.onItemContainerChanged(new ItemContainerChanged(InventoryID.EQUIPMENT.getId(), itemContainer));
+		String actualString = prayerPlugin.getEstimatedTimeRemaining(true);
+
+		Assert.assertEquals(expectedString, actualString);
```

Comment:
Import `assertEquals`

Reply:
Imported `assertEquals` in the latest commit